### DISCUSSION TITLE
Anything I can help with?

### DISCUSSION THREAD

**Discussion by the-bort-the** [9p4/jellyfin-plugin-sso#225] [Q&A]
Is there anything you could use in terms of help for the project? I really like it and want to participate if possible.

**Comment by 9p4** [comment]
I'd love some PRs that split apart logic and the controllers and make the code much more maintainable.

**Reply by the-bort-the** [reply]
If you have something of focus, I'll take a look there first. I have more of a TypeScript skillset, but I'm down to learn C#.

**Reply by 9p4** [reply]
First split up this file: https://github.com/9p4/jellyfin-plugin-sso/blob/main/SSO-Auth/Api/SSOController.cs

**Reply by 9p4** [reply]
Also I'd love for some automated testing.
